### DISCUSSION TITLE
does paint dark mode work on windows 10?

### DISCUSSION THREAD

**Discussion by geextahslex** [ramensoftware/windhawk#193] [Q&A]
Hi, I want to know if the paint dark mode works on windows 10? Or is it meant for Win 11 only?

Thank you :)

**Comment by m417z** [comment]
Not with the Win10 Paint, but as far as I know the Win11 Paint can be installed on Win10 and then it works. But the latest Paint versions support dark mode natively, so I'm not sure how useful this mod still is unless you're stuck with an old Win11 version.
@ahmed605 I think it's best to add information about this to the readme, that's not the first time that such questions come up.

**Reply by ahmed605** [reply]
I don't think the mod is useful anymore, I will make a PR removing it as soon as I can

**Reply by m417z** [reply]
Mods aren't removed from the repository unless there's some extraordinary case such as a malware or illegal content. Also, maybe some users still want to use it. I think it's best to add a clear disclaimer that explains that it's only useful for old Win11 Paint versions.

**Comment by geextahslex** [comment]
@ahmed605 I managed to install Paint 11 on Win 10 but it's bugged. Do you know how to deal with it? I think it's transparent where Mica should be.
![paint bug](https://github.com/ramensoftware/windhawk/assets/139760266/47b94181-2946-45b9-b3fa-53b39d448bbc)

**Reply by ahmed605** [reply]
I think you can use [Mica For Everyone](https://github.com/MicaForEveryone/MicaForEveryone) to replace it with Acrylic or something

**Reply by geextahslex** [reply]
I don't want mica xD I need an updated guide for this. All guides are 2 years old and don't work properly. 
I couldn't install microsoft.ui.xaml.2.8.6, do you think this could be the reason?

**Reply by ahmed605** [reply]
Mica For Everyone can do Acrylic too afaik, not just Mica
